lib/ReactViews/Mobile/MobileHeader.jsx

```diff
@@ -167,7 +167,11 @@ const MobileHeader = observer(
                 <Box
                   position="absolute"
                   css={`
-                    left: 5px;
+                    right: 270px;
+                    [dir="rtl"] & {
+                      left: 270px;
+                      right: 0px;
+                    }
```

Comment:
This wasn't ideal in LTR or RTL. I've substituted wrapping MobileHeader in `stylis-plugin-rtl` which seems to work well.